### PR TITLE
create periodic elmental 2048

### PR DESCRIPTION
**Pull request by Griffintaur**
adding another project.

### CODE REVIEW

File: _data/projects/Periodic-Elemental-2048.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+name: Periodic-Elemental-2048
+desc: Periodic Elemental  2048 with tweak where elements combine to give higher elements
+site: https://github.com/Griffintaur/Periodic-Elemental-2048/
+tags:
+- oss
+- javascript
+upforgrabs:
+- name: help wanted
```
Comment (ritwik12) [inline _data/projects/Periodic-Elemental-2048.yml]: @Griffintaur name and link should not have "-" in front of them.

Comment (Griffintaur) [reply _data/projects/Periodic-Elemental-2048.yml]: Thanks @ritwik12

**Comment by Griffintaur** [comment]
@ritwik12  task is already defined in the issue section.

**Comment by ritwik12** [comment]
@Griffintaur With that label I meant changes required.

**Comment by ritwik12** [comment]
@Griffintaur I can see that your project is a fork of someone's project, are you sure that you want to move with a fork? If the original repo owner have no problem with duplicate then you could move your fork to new repo also. You can easily do that by duplicating repo. Tell me whatever way you feel good.

**Comment by ritwik12** [comment]
@Griffintaur You can simply do this by creating a new repo. While creating new repo don't select to add readme.md and create a blank repo, then you can see import code from another repo at the bottom of new repo. That way you'll have duplicate of your fork.
If you want to do it on command line please follow - https://help.github.com/articles/duplicating-a-repository/

**Comment by Griffintaur** [comment]
@ritwik12. I will create a new repo and move code there. Thanks

**Comment by Griffintaur** [comment]
closes #942

**Comment by Griffintaur** [comment]
@ritwik12 . I have created another repo for this project. Please have a look.

**Comment by ritwik12** [comment]
@Griffintaur Looks good now.

**Comment by ritwik12** [comment]
@Griffintaur Thanks!!

**Pull request merged**